artifact-manifest/artifact-manifest.md

```diff
@@ -0,0 +1,462 @@
+# OCI Artifact Manifest
+
+The OCI artifact manifest provides a means to define a wide range of artifacts, including a chain of dependencies of related artifacts. It provides a means to define multiple collections of types, including blobs, dependent artifacts and referenced artifacts.
```

Comment:
My working assumption how SBoMs would be represented in a registry would be a new artifact, with a dependency to the thing artifact it's declaring it's an SBoM of. Essentially, it would be the same as a Notary v2 signature. 
The SBoM artifact would have its own `blobs`, optionally a config, and a dependency (manifest entry in the latest example). It _would not_ have a tag, has the tag of an SBoM isn't really interesting as the SBoM is an "enhancement" to an existing tagged artifact in a repo.